src/tests/Microsoft.PowerFx.Interpreter.Tests/LanguageServiceProtocol/LanguageServerTests.cs

```diff
@@ -913,12 +1078,15 @@ public void TestGetPosition()
         }
 
         [Theory]
-        [InlineData("Power(", 6, "Power(2,", 8, false)]
-        [InlineData("Behavior(); Power(", 18, "Behavior(); Power(2,", 20, true)]
+        [InlineData("Power(", 6, "Power(2,", 8, false, true)]
```

Comment:
>true [](http://example.com/codeflow?start=55&length=4)

Ditto

Reply:
What's wrong in writing a theory test? After it was meant for cases like this. Both "true" and "false" are mapped to different tests so we can identify which one failed since both are independent of each other? Not a fan of creating "sub tests" within a test by looping over test cases. If it was the recommended way then all the InlineDatas would be converted to a list of test cases and a loop would be to execute them one by one. Wherever possible, i prefer to use InlineData and write data driven tests. Anyways, i am fine with using loops here.

Reply:
Refactored